.github/workflows/test-ci-bootstrap.yml

```diff
@@ -32,6 +32,8 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       - name: Set up Terraform
         uses: hashicorp/setup-terraform@v3
+        with:
+          version: "1.7.5" # Pin until 1.8.x crash has been resolved
```

Comment:
Does this need to be `terraform_version`?

Reply:
Yes!